UDPRenderer.pde

```diff
@@ -0,0 +1,59 @@
+import hypermedia.net.*;
+
+class UDPRenderer {
+  
+  private UDP udp;
+  private Config config;
+  
+  private Grid grid;
+  
+  UDPRenderer(Config config) {
+    this.config = config;
+    
+    udp = new UDP(this, config.localPort);
+    //udp.log(true);     //printout the connection activity
+    udp.listen(true);
```

Comment:
There are no messages expected to be received as far as I know. Is this required by the UDP library to send out packets?